travis/script.sh

```diff
@@ -200,6 +200,15 @@ echo "Running test ${TEST}"
       echo "Docker compose container status"
       docker-compose -f docker-compose.yml ps
       ;;
+    unittests)
+      echo "run unittests scripts"
+      # change user id withn Docker container to user id of travis user
+      sed -i -e "s/USER\ 1001/USER\ `id -u`/g" ./Dockerfile.django
+      cp ./dojo/settings/settings.dist.py ./dojo/settings/settings.py
+      source ./docker/setEnv.sh release
```

Comment:
this line is not required as it's the default mode

Reply:
line 206 helps me solve the challenge i hv been having for several months now...error from permission..

line 207 and 206 seems like it's not needed now..but when i was debugging error ..release mode does not tell me what's wrong..but i used dev mode then. With dev mode i was able to debug and trace my issue....It's when i fixed it..i now returned it back to release...
..some one may need to change it back to dev in future when they are trying to debug why their test fails....that's why i would suggest we let it be...

Reply:
it won't really do anytn now since the container is already in release mode..but it would be cool to know that one can change the mode at anytime....someone may want to run the test that's available only to unittest mode..